tools/release/cloudbuild.yaml

```diff
@@ -120,14 +111,31 @@ steps:
 ## Tag all of the images as "commit-latest-master-build" so that other processes
 ## can easily pick up the latest one.
 - name: gcr.io/cloud-builders/docker
-  args: ["tag", "gcr.io/cloud-datalab/datalab:commit-${REVISION_ID}", "gcr.io/cloud-datalab/datalab:commit-latest-master-build"]
-  waitFor: ["buildDatalab"]
+  args: ['tag', 'gcr.io/cloud-datalab/datalab:commit-${REVISION_ID}', 'gcr.io/cloud-datalab/datalab:commit-latest-master-build']
+  waitFor: ['buildDatalab']
 - name: gcr.io/cloud-builders/docker
-  args: ["tag", "gcr.io/cloud-datalab/datalab-gpu:commit-${REVISION_ID}", "gcr.io/cloud-datalab/datalab-gpu:commit-latest-master-build"]
-  waitFor: ["buildDatalabGPU"]
+  args: ['tag', 'gcr.io/cloud-datalab/datalab-gpu:commit-${REVISION_ID}', 'gcr.io/cloud-datalab/datalab-gpu:commit-latest-master-build']
+  waitFor: ['buildDatalabGPU']
 - name: gcr.io/cloud-builders/docker
-  args: ["tag", "gcr.io/cloud-datalab/datalab-gateway:commit-${REVISION_ID}", "gcr.io/cloud-datalab/datalab-gateway:commit-latest-master-build"]
-  waitFor: ["buildGateway"]
+  args: ['tag', 'gcr.io/cloud-datalab/datalab-gateway:commit-${REVISION_ID}', 'gcr.io/cloud-datalab/datalab-gateway:commit-latest-master-build']
+  waitFor: ['buildGateway']
+
+## Tag the config local file with latest-master-build as well.
+- name: 'gcr.io/cloud-builders/gsutil'
+  args: ['cp',
+         '/workspace/config_local_${REVISION_ID}.js',
+         'gs://${PROJECT_ID}/builds/config_local_latest-master-build.js']
+  id:   'pushConfigLocalStatic'
+  waitFor: ['pushConfigLocal', 'buildDatalab', 'buildDatalabGPU', 'buildGateway']
+
+# Cleanup the temporary files created to push build artifacts to GCS
+- name: 'debian'
+  args: ['rm', '-rf',
+         '/workspace/config_local.js',
+         '/workspace/config_local_${REVISION_ID}.js',
+         '/workspace/datalab-cli-${REVISION_ID}.tgz']
+  id:   'cleanupWorkspace'
+  waitFor: ['pushConfigLocal', 'pushTarball', 'pushConfigLocalStatic']
```

Comment:
This was intentionally being done prior to the Docker builds so that we would not have any unwanted artifacts that could accidentally get copied into one of the Docker images.

Instead of moving it down here, just make the `gsutil` command copy directly from one location in the bucket to another (i.e. you make both the source and target remote files.